routes/common/load_balancer_helper.rb

```diff
@@ -119,6 +120,31 @@ def delete
     @request.halt
   end
 
+  def download_certificate_bundle
```

Comment:
hmmm do we actually want to ship this feature, to get frequently-rotating private keys out via the web page? I understand why that may have been useful in development, but who is going to be clicking around every thirty or ninety days to do this?

Reply:
You're right. At the time, I didn't think I would be this quick to implement the certificate server and I wanted to give a way to customer to get the certificates. However, considering we have the pr #1858 out, I'm removing this.